packages/kununu-header/Header.jsx

```diff
@@ -11,25 +11,53 @@ export default function Header ({
   assetsPath,
   title,
   logoLink,
+  simple,
 }) {
+  const simpleLogo = () => {
+    const content = (
+      <picture>
+        <source
+          srcSet={`${assetsPath}/logo-desktop.svg`}
+        />
+        <img
+          alt={title}
+          aria-hidden="true"
+          className={styles.simpleLogo}
+          rel="presentation"
+          src={`${assetsPath}/logo-desktop.svg`}
+        />
+      </picture>
+    );
+
+    return logoLink ?
+      React.cloneElement(logoLink, logoLink.props, content) :
+      content;
+  };
+
   return (
     <header
       role="banner"
-      className={`${styles.header} ${fixed && styles.fixed}`}
+      className={`${styles.header} ${fixed && styles.fixed} ${simple && styles.simple}`}
     >
       <div className={container}>
         <div className={styles.flex}>
           <div className={styles.pullLeft}>
-            <Logo
-              link={logoLink}
-              assetsPath={assetsPath}
-              title={title}
-            />
+            {simple ? simpleLogo() : (
+              <Logo
+                link={logoLink}
+                assetsPath={assetsPath}
+                title={title}
+              />
+            )}
+
             <span className={styles.title}>{title}</span>
           </div>
-          <div className={styles.pullRight}>
-            {children}
-          </div>
+          {!simple ? (
```

Comment:
maybe `&&`?

Reply:
ok, I guess I will just switch to always using && then 😆 Until someone requests ? : again

Reply:
Daniel ended up refactoring this and it's much prettier now 🌷 so this can be left